model/settings/init.go

```diff
@@ -8,6 +8,7 @@ import (
 var service *SettingsService
 
 type Service interface {
+	PublicName(db prefixer.Prefixer) (string, error)
```

Comment:
Maybe use the same naming convention for the prefixer as the other methods of the service (i.e. `inst`)?

Reply:
Could you explain why you decided to move this method here? I'm curious.

Reply:
This logic is part of the Settings as the data is stored inside the instance `settings` database. 

>  Maybe use the same naming convention for the prefixer as the other methods of the service (i.e. inst)?

Yep, didn't made my mind on it yet. I have started with `inst` but it appears that the rest of the codebase use `db` so I will make the changes to follow the convention.